docs/usage/delete.rst.inc

```diff
@@ -112,4 +112,5 @@ two options are mutually exclusive.
 
 To avoid accidentally deleting archives, especially when using glob patterns,
 it might be helpful to use the ``--dry-run`` to test out the command without
-actually making any changes to the repository.
\ No newline at end of file
+actually making any changes to the repository.
```

Comment:
maybe do git checkout thatfile to revert it to the original state so it disappears from the PR.